File: src/main/webapp/app/model/CalendarEvent.js
/**
 * Model::CalendarEvent
 * 캘린더 일정 항목을 정의한다.
 */
Ext.define('CALHUB.model.CalendarEvent', {
    extend: 'Ext.data.Model',
    fields:
        [
            {
                name: 'uid',
                type: 'string'
            },{
                name: 'summary',
                fields: [
                    {name: 'name', type: 'string'},
                    {name: 'value', type: 'value'}
                ],
                convert: function(v, record) {
                    return v.value !== undefined ? v.value : v;
                }
            },{
                name: 'dateStamp',
                fields: [
                    {name: 'date', type: 'int'},
                    {name: 'dateTime', type: 'int'},
                    {name: 'name', type: 'string'},
                    {name: 'value', type: 'string'}
                ],
                convert: function(v, record) {
                    return v.date !== undefined ? v.date : v;
                }
            },{
                name: 'organizer',
                fields: [
                    {name: 'name', type: 'string'},
                    {name: 'value', type: 'string'},
                    {name: 'calAddress', type: 'string'},
                ],
                convert: function(v, record) {
                    return v.value !== undefined ? v.value : v;
                }
            },{
                name: 'location',
                type: 'string',
                convert: function(v, record) {
                    return v == null ? '' : ( v.value !== undefined ? v.value : v );
                }
            },{
                name: 'startDate',
                fields: [
                    {name: 'name', type: 'string'},
                    {name: 'date', type: 'int'},
                    {name: 'value', type: 'string'}
                ],
                convert: function(v, record) {
                    return v.date !== undefined ? v.date : v;
                }
            },{
                name: 'endDate',
                fields: [
                    {name: 'name', type: 'string'},
                    {name: 'date', type: 'int'},
                    {name: 'value', type: 'string'}
                ],
                convert: function(v, record) {
                    return v.date !== undefined ? v.date : v;
                }
            },
            'description'
        ]
});

File: src/main/java/com/idatabank/bisu/calhub/model/CalendarEvent.java
package com.idatabank.bisu.calhub.model;

import java.util.Date;

public class CalendarEvent {
    
    private String uid = null;
    private String summary = null;
    private Date dateStamp = null;
    private String organizer = null;
    private String location = null;
    private Date startDate = null;
    private Date endDate = null;
    private String description = null;
    public String getUid() {
        return uid;
    }
    public void setUid(String uid) {
        this.uid = uid;
    }
    public String getSummary() {
        return summary;
    }
    public void setSummary(String summary) {
        this.summary = summary;
    }
    public Date getDateStamp() {
        return dateStamp;
    }
    public void setDateStamp(Date dateStamp) {
        this.dateStamp = dateStamp;
    }
    public String getOrganizer() {
        return organizer;
    }
    public void setOrganizer(String organizer) {
        this.organizer = organizer;
    }
    public String getLocation() {
        return location;
    }
    public void setLocation(String location) {
        this.location = location;
    }
    public Date getStartDate() {
        return startDate;
    }
    public void setStartDate(Date startDate) {
        this.startDate = startDate;
    }
    public Date getEndDate() {
        return endDate;
    }
    public void setEndDate(Date endDate) {
        this.endDate = endDate;
    }
    public String getDescription() {
        return description;
    }
    public void setDescription(String description) {
        this.description = description;
    }
}